flavoUr/settings.py

```diff
@@ -55,10 +55,12 @@
 
 ROOT_URLCONF = 'flavoUr.urls'
 
+PROJECT_PATH = os.path.realpath(os.path.dirname(__file__))
+
```

Comment:
ok so you can take this line out because we already have something build into django that is kind of similar. Look in this file up on line 16 and you'll see BASE_DIR. It's basically the same thing you did here except one level up. So if we take this line out, we can put the following on line 126:

```os.path.join(BASE_DIR, 'flavoUr/static')```